### DISCUSSION TITLE
Output jupyter-like outputs into a box or container.

### DISCUSSION THREAD

**Discussion by armandophys** [beeware/toga#1319] [Q&A]
I've only jumped on the toga train yesterday but I was wondering if there was a way to display outputs like is done in jupyter notebooks. This would be elements such as a static graph, latex formatted output, and maybe even a dynamic graph such as plotly. 

Based on what I've read, I bet plotly could be done but I'd like to just be able to plot a graph and maybe put a latex output next to it. I did see toga-chart but there are no examples so I'm not sure if this is something that is specific to matplotlib or can be used with any graphical output such as a sympy latex output.

**Comment by freakboy3742** [comment]
It depends on exactly what features you're looking for. There is [toga-chart](https://github.com/beeware/toga-chart), which is a matplotlib rendering backend that draws on a Toga canvas - so you can do charting in Toga. 

However, we don't have a LaTeX output formatter, or a plotly plugin. I'm not opposed to adding them (or supporting them as plugins), but they're not there "out of the box" today.
